examples/3d/elixir_euler_ec.jl

```diff
@@ -0,0 +1,55 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible ideal GLM-MHD equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = Trixi.initial_condition_weak_blast_wave
+
+surface_flux = flux_ranocha
+volume_flux  = flux_ranocha
+solver = DGSEM(3, surface_flux, VolumeIntegralFluxDifferencing(volume_flux))
+
+coordinates_min = (-2, -2, -2)
+coordinates_max = ( 2,  2,  2)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=3,
+                n_cells_max=10_000)
+
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver)
+
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+stepsize_callback = StepsizeCallback(cfl=1.4)
+
+save_solution = SaveSolutionCallback(interval=10,
```

Comment:
```suggestion
save_solution = SaveSolutionCallback(interval=100,
```
In `examples/3d/parameters_euler_ec.toml`, we have `solution_interval = 100`.